using_the_compiler/README.md

```diff
@@ -44,6 +44,25 @@ Make sure to always use `--release` for production-ready executables and when pe
 
 The reason for this is that performance without full optimizations is still pretty good and provides fast compile times, so you can use the `crystal` command almost as if it were an interpreter.
 
+To reduce the binary size, you can add the `--no-debug` flag and use the `strip` command. Debug symbols will be removed, use this options if only the size is an issue and you won't need to debug the program.
+
+## Creating a statically-linked executable
+
+This following libraries need to be required in the programm to force the linking against `libstdc++`:
+
+```
+require "llvm/lib_llvm"
+require "llvm/enums"
```

Comment:
Very heavily -1 on documenting a bug.

Reply:
I guess we will add this to the wiki, I remove it.